how-we-work/all-hands-meeting.html.md

```diff
@@ -1,14 +1,39 @@
 ---
 title: All Hands meeting
 ---
-Every month, we hold an All Hands meeting. These meetings are used to:
-
-- give a general update on the status of Nebulab;
-- introduce new team members;
-- present any important updates to the playbook.
-
-You are strongly encouraged to attend so that you can ask questions and spend some virtual
-bonding time with the entire team, but in case you're not able to, the meetings are recorded.
+Every month, we hold a company-wide meeting, called the All Hands meeting. We use this
+session to provide useful information to every team member, with the intent of being as much
+transparent as possible about the direction of Nebulab, along with its successes, failures
+and next things to do. Having everyone on the same page is our way to try to drive every
```

Comment:
I would say "keeping" everyone on the same page, since it's an active effort on our end.